wifiphisher/common/phishinghttp.py

```diff
@@ -15,59 +16,60 @@
 
 class DowngradeToHTTP(tornado.web.RequestHandler):
 
-    def get(self, url):
+    def get(self):
         self.redirect("http://10.0.0.1:8080/")
 
 
-class ShowPhishingPageHandler(tornado.web.RequestHandler):
+class CaptivePortalHandler(tornado.web.RequestHandler):
 
-    def get(self, url):
+    def get(self):
+        client_request = self.request.path[1:]
+        render_file = "index.html"
         try:
-            if "/" in url:
-                self.redirect("/index")
-            self.render("index.html", **template.get_context())
+            if self.request.path == "/":
```

Comment:
This would leave requests sent to the root folder (http://10.0.0.1:8080/) unanswered, we need to keep a check in place for paths that are root "/"

We also want to include a check to make sure special requests (not root), were referred by our host before sending the user to a page within our scenario, otherwise a user just connected to our AP requesting http://randomsite.com/logout.html would get redirected to the `logout.html` page of our captive portal without actually having visited the index page first. This info can be found in the headers of the request.